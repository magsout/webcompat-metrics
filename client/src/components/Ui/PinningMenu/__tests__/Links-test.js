/* This Source Code Form is subject to the terms of the Mozilla Public
 * License, v. 2.0. If a copy of the MPL was not distributed with this
 * file, You can obtain one at http://mozilla.org/MPL/2.0/. */

import React from "react";
import renderer from "react-test-renderer";

import Links from "../Links";

it("renders Links default correctly", () => {
  const tree = renderer
    .create(
      <Links>
        <span>content</span>
      </Links>,
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
