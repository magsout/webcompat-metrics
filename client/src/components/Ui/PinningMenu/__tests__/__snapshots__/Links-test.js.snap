// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Links default correctly 1`] = `
<nav
  className="links"
  role="navigation"
>
  <span>
    content
  </span>
</nav>
`;
